src/components/EditDeliveryDetailsPopup/EditDeliveryDetailsPopup.tsx

```diff
@@ -0,0 +1,110 @@
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { resetDeliveryDetailsState } from 'redux/deliveryDetailsSlice';
+import { hidePopup, selectEditDeliveryDetailsPopup } from 'redux/popupSlice';
+import { useAppSelector } from 'redux/store';
+import Button from 'components/Button';
+import InnerPopupWrapper from 'components/InnerPopupWrapper';
+import RecipientForm from 'components/DeliveryDetails/RecipientForm';
+import { ReactComponent as CheckmarkIcon } from 'assets/images/greenCheckmark.svg';
+import {
+  ButtonsStyled,
+  ContentStyled,
+  HeaderStyled,
+  InfoTextStyled,
+  ThankYouPageStyled
+} from './EditDeliveryDetailsPopupStyled';
+
+const EditDeliveryDetailsPopup = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+
+  const { offerId, offerTitle } = useAppSelector(
+    selectEditDeliveryDetailsPopup
+  );
+
+  const [currentStep, setCurrentStep] = useState(1);
+
+  useEffect(() => {
+    return () => {
+      dispatch(resetDeliveryDetailsState());
+    };
+  }, []);
+
+  return (
+    <InnerPopupWrapper
+      steps={2}
+      isError={false}
+      currentStep={currentStep}
+      popupTitle={t(
+        'edit-delivery-details-popup.title',
+        'Edit Delivery Details'
+      )}
+    >
+      <ContentStyled>
+        {currentStep === 1 ? (
+          <>
+            <HeaderStyled>
+              {t('edit-delivery-details-popup.header', 'Edit Delivery Details')}
+            </HeaderStyled>
+            <InfoTextStyled>
+              {t(
+                'edit-delivery-details-popup.info-text-1',
+                'You have selected the issue regarding your'
```

Comment:
are these copies form the design or Anna? If from design, do we have a confirmation for Anna for all copies?

Reply:
Nope, will confirm then. But in future it would be nice if finished Figma design wouldn't need copy confirmation.

Reply:
copies confirmed and updated, also will be different when gift is already sent